rules_poetry/poetry.bzl

```diff
@@ -79,8 +94,31 @@ def _mapping(repository_ctx):
     return {
         "dependencies": dependencies,
         "groups": groups,
+        "pyproject": pyproject
     }
 
+def extract_markers(repository_ctx, resolved_markers, dep, markers):
+    python_interpreter = _resolve_python_interpreter(repository_ctx)
+
+    if str(type(markers)) == 'list':
+        if dep not in resolved_markers:
+            resolved_markers[dep] = {}
+        for marker in markers:
+            for platform in SUPPORTED_PLATFORMS:
+                system, machine = platform.split('!')
+                test_string = marker['markers'].replace('platform_machine', "'" + machine + "'")
+                test_string = test_string.replace('sys_platform', "'" + system + "'")
+                cmd = [
+                    python_interpreter,
+                    '-c',
+                    'print(' + test_string + ')'
+                ]
+                result = repository_ctx.execute(cmd)
+                if result.stdout.strip() == "True":
```

Comment:
Nit: maybe use `str(True)` just to make it super clear you're evaluating the Python interpreter's output?

Reply:
fixed in [7aa2a09](https://github.com/mongodb-forks/rules_poetry/pull/1/commits/7aa2a09847cc9482b937f4381021a9cb45946821)